src/main.rs

```diff
@@ -113,6 +119,9 @@ static REDIS_POOL: Lazy<Pool<RedisConnectionManager>> = Lazy::new(|| {
     let manager = RedisConnectionManager::new(redis_cnxn_url).unwrap();
     Pool::builder().build(manager).unwrap()
 });
+static USE_EXCHANGE: Lazy<bool> = Lazy::new(|| LOCAL_CONF.get("use_exchange").unwrap_or(false));
```

Comment:
Is USE_EXCHANGE for the user withdraw flow from the exchange or the Interexchange settlement?

Reply:
Withdraw for now